rancher/rancher-compose/upgrading/index.md

```diff
@@ -69,6 +69,21 @@ By default, during the upgrade, there is a 2 second wait between when containers
 $ rancher-compose upgrade service1 service2 --interval "30000"
 ```
 
+#### Starting New Containers Before Stopping Old Containers
+
+By default, the in-service upgrade will stop the existing containers, and then launch the new containers. If you'd like to start the new containers before stopping the old containers, you would provide additional content in the `rancher-compose.yml` that would be used for the in-service upgrade.
```

Comment:
My two cents:

By default, the in-service upgrade terminates the existing containers, and then launch the new containers. To start a new container before stopping the old containers, you must provide additional content in the 'rancher-compose.yml' file that is used for the in-service upgrade.